core/services/fluxmonitor/flux_monitor.go

```diff
@@ -15,14 +9,21 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/ethereum/go-ethereum/common"
+	"chainlink/core/eth"
+	"chainlink/core/eth/contracts"
+	"chainlink/core/logger"
+	"chainlink/core/store"
+	"chainlink/core/store/models"
+	"chainlink/core/store/orm"
+	"chainlink/core/utils"
+
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/pkg/errors"
 	"github.com/shopspring/decimal"
 	"go.uber.org/multierr"
 )
 
-//go:generate mockery -name FluxMonitor -output ../../internal/mocks/ -case=underscore
+//go:generate mockery -name Service -output ../../internal/mocks/ -case=underscore
```

Comment:
Shouldn't this be FluxMonitorService?  Another route we could go is generate the Mock in the package

Reply:
It should, but I couldn't figure out how to force mockery to give the actual type a different name.  The `-name` flag seems to create a hard 1:1 association between the mocked type's name and the mock's name.  I'd really like to deal with this one, though, so let me know if I'm missing anything.  Otherwise we might have to put mocks into subfolders of `mocks/` with names based on their original packages.